pkg/irreparabledb/irreparabledb_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparabledb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/internal/testcontext"
+)
+
+func TestIrreparableRepository(t *testing.T) {
+	//testing variables
+	TestRmtSegInfo := RemoteSegmentInfo{
+		SegmentPath:        []byte("IamSegmentkeyinfo"),
+		SegmentDetail:      []byte("IamSegmentdetailinfo"),
```

Comment:
`SegmentDetail` is still unclear. What type does it contain, is it a string, something else?

Reply:
segment detail has pb.Pointer{} info

pointer := &pb.Pointer{}
err = proto.Unmarshal(item.Value, pointer)

Reply:
I'd like to see this in the RemoteSegmentInfo declaration.